lib/api.js

```diff
@@ -882,6 +895,7 @@ function makeWrappedRunner (run, callback) {
  * @param {object}  [opts] - options
  * @param {boolean} [opts.enabled=true] - enable tracing
  * @param {boolean} [opts.collectBacktraces=false] - collect backtraces
+ * @param {boolean} [opts.forceNewTrace] - force a new trace, ignoring any existing context (but not xtrace)
  * @param {string|function} [opts.customTxName] - name or function
  * @returns {value} the value returned by the run function or undefined if it can't be run
```

Comment:
There is also a 'cb' parameter, should it be mentioned here

I'm also wondering why the test on line 941 is `if (typeof opts !== 'object')` and not a check if opts is a function.

Reply:
because both opts and cb could be `undefined` and `undefined` is not an object. javascript has a few quirky things like that.

Reply:
add cb documentation too. i think it was there at one point but obviously not now.